### DISCUSSION TITLE
Icelandic language leader

### DISCUSSION THREAD

**Discussion by snjallingur** [OHF-Voice/intents#420] [Language Leaders]
Hi,
I am running a service company in Iceland (snjallingur.is) that is focussed on providing services for homes and smaller companies with Home Assistant in Iceland. We are also administrators of the Home Assistant facebook group in Iceland.
I would be more than happy to assist and help out with adding voice assistance for the Icelandic language. I already have been in contact with local companies like grammatek.is and mideind.is to leverage available STT and TTS APIs.
Would be great to be able to give something back to the community.
Hope for a positive response.
BR

**Comment by synesthesiam** [comment]
Thanks @snjallingur! I will create some empty Icelandic templates so you can get started :+1: 

I have seen a lot of speech data for Icelandic available oh [OpenSLR](http://www.openslr.org/resources.php). Would you be interested in working together to create some STT/TTS models?

**Reply by snjallingur** [reply]
Thanks, I will start with the basic templates within the next days.
And yes I can possibly help creating/modifying STT/TTS models. I will at least contact people to help out in this area.
BR Alexander

**Reply by Daviderikmollberg** [reply]
Hey @synesthesiam we in Tiro (https://tiro.is/) can possible help with the STT/TTS part. Can you tell me more about what type of models you are expecting and how they are hosted?

**Reply by synesthesiam** [reply]
Hi @Daviderikmollberg, I prefer models that can be self-hosted (usually on the same device running Home Assistant). For STT, I go for Kaldi or [Coqui STT](https://stt.readthedocs.io/en/latest/). For TTS, I have [Larynx](https://github.com/rhasspy/larynx2/) which is just a VITS model exported to the Onnx runtime.
